hack/k8s/helm/nuclio/README.md

```diff
@@ -63,37 +63,42 @@ helm install \
 ```
 
 ### Install on Minikube using a local, insecure registry
-By clearing `registry.secretName`, Nuclio will not try to load Docker secrets.
+
+You will need to run a local Docker registry. Run the following command on the host if you're working with Docker for Mac or on the Minikube VM:
+```sh
+docker run -d -p 5000:5000 registry:2
```

Comment:
Maybe worth mentioning to add `localhost:5000` to insecure registries

Reply:
Works without explicitly adding it